src/hooks/useMounted.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useRef } from 'react';
 
-export const useMounted = () => {
-  const isMounted = useRef(false);
+export const useMounted = (): boolean => {
+  const isMounted = useRef<boolean>(false);
```

Comment:
It is also not required to set ```<boolean>``` type here, you provided default value and TS with use type from the default value =)

Reply:
It is a redundant code. I have fixed it!